src/ext/components/frames-output.pug

```diff
@@ -45,6 +45,7 @@
           // ko ifnot:isText
           p
             +button('Build Model&hellip;', 'cube', 'createModel')
+            +button('Run AutoML', 'sitemap', 'createAutoML')
```

Comment:
The ellipsis makes sense for `Build Model...` because you still have to select which model later, no such a thing with `AutoML`.
Somehow, it means `Build Model GBM`, `Build Model GLM`, ...

Reply:
Or it could also mean `Build Model with the frame`.
I have to admit that the meaning of this ellipsis is not clear.

Reply:
My perception of the ellipsis is "(continue with) Build Model". As you can see Inspect is the only in this group that does not have ..., thats because it does not build anything new, things that produce something new (predictions, model) have the ...
at least thats my perception of it

Reply:
it might be best just to save realestate and get rid of all the hellips

Reply:
yeah, removed them, they were inconsistent and not very useful.